src/api/app/models/project.rb

```diff
@@ -79,6 +79,7 @@ def autocomplete(search)
 
   has_one :staging, class_name: 'StagingWorkflow', inverse_of: :project
   belongs_to :staging_workflow, inverse_of: :staging_projects
+  has_many :staged_requests
```

Comment:
Now we're talking. So are these the requests in the target project that are staged or are these the requests selected into a staging project?

Reply:
@coolo:

> Now we're talking. So are these the requests in the target project that are staged or are these the requests selected into a staging project?

The first, the requests in the target project that are staged.

Anyway, this is still not finished. Marking the pull request as WIP.